packages/material-ui/src/ButtonBase/ButtonBase.test.js

```diff
@@ -374,6 +379,27 @@ describe('<ButtonBase />', () => {
         ).to.have.lengthOf(0);
       });
 
+      it('should stop the ripple when the context menu opens', () => {
+        const { getByRole } = render(
+          <ButtonBase
+            TouchRippleProps={{
+              classes: {
+                rippleVisible: 'ripple-visible',
+                child: 'child',
+                childLeaving: 'child-leaving',
+              },
+            }}
+          />,
+        );
+        const button = getByRole('button');
+        fireEvent.contextMenu(button);
```

Comment:
The test case seems to not be failing on HEAD (I didn't try). Shouldn't we trigger the ripple first? 

```suggestion
        fireEvent.mouseDown(button);
        fireEvent.contextMenu(button);
```

Reply:
I was thinking about what would be the better approach. This seems the correct way. I'll update it.

Reply:
Updated